<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_deal</class>
 <widget class="QTabWidget" name="student_deal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-size:24px;
font-family:Century Gothic;
}



QTabBar::tab {
		border: none;
		border-top-left-radius: 4px;
		border-top-right-radius: 4px;
        border-top-color:black;
		color: black;
	    border-top:1px solid #ff0000;
        border-left:1px solid #ff0000;

		height: 120px;
		min-width: 60px;
		margin-right: 5px;
		padding-left: 5px;
		padding-right: 5px;
}
QTabBar::tab:hover {
		background: rgb(0, 78, 161);
}
QTabBar::tab:selected {
		color: white;
		background: rgb(0, 78, 161);
}
#tab_2{border-image: url(:/new/prefix1/背景8.png);border-image:}

#tab{border-image: url(:/new/prefix1/背景7.png);}
#tab_3{border-image: url(:/new/prefix1/背景9.png);}












</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::West</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string> 我的信息</string>
   </attribute>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>501</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background:transparent;
border-radius:15px;
border:2px solid #000000;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>451</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string> 姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="styleSheet">
           <string notr="true">
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string> 性别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string> 男</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string>女</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string> 邮箱</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_email">
          <property name="styleSheet">
           <string notr="true">
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string> 年龄</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string>修改密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pwd">
          <property name="styleSheet">
           <string notr="true">
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color:black;
background:transparent;
border:transparent;</string>
          </property>
          <property name="text">
           <string>确认密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pwd1">
          <property name="styleSheet">
           <string notr="true">
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>450</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
background:black;
border-radius:15px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
background:#333;
border-radius:15px;
background:#49ebff;
}
</string>
     </property>
     <property name="text">
      <string>确 认 完 善</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>              预约课程               </string>
   </attribute>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>301</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  stop: 0 #dadbde, stop: 1 #f6f7fa);
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>点击预约</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>171</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>教练查询:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>441</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>预约情况</string>
   </attribute>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  今天</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>  明天</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>231</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 时间段：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  教练：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>420</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  教练：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>280</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  stop: 0 #dadbde, stop: 1 #f6f7fa);
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>查看教练信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  stop: 0 #dadbde, stop: 1 #f6f7fa);
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>查看教练信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>111</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>   时间段：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
